handbook/communicating.md

```diff
@@ -0,0 +1,52 @@
+# Other communication
+
+:::{admonition} NEW!!!
+:class: warning
+This content is new and should be reviewed!
+:::
+
+GitHub is a fantastic platform for collaborating on open source software, but it can't do *everything*.  Additionally, we work with other people or institutions that use *different* platforms for collaboration, teamwork, and communication.  While Xdev uses GitHub for as much as we can, we acknowledge that sometimes you have to use something other than GitHub from time to time.
+
+## Email
+
+Email should be used sparingly, but there are times when it is necessary.  Sending invitations to Zoom or Google Meet calls sometimes requires knowing a teammate's individual email address, which results in an email notification of the newly scheduled event.  Email is also a convenient mechanism for people *outside* of the team to communicate with us, in which case responses over email are obviously appropriate.
+
+Communication directly between team members with email should be discouraged, though.  The only time when such communication is appropriate is when you need to reference another email (e.g., replying or forwarding) for discussion, or you need to reference a Zoom or Google Meet event for which you were notified via email.
+
+These rules apply for both your individual work email addresses *and* the Xdev Team email address (`xdev@ucar.edu`).
+
+## Zulip
+
+[Zulip](https://zulip.com) is an open source chat platform for distributed teams.  Because it is open source, we were able to launch our own Zulip server on NCAR systems.  We launched this platform primarily as a means for scientists to collaborate with the Xdev Team *and each other* around issues related to the adoption of Python and the Pangeo stack.
+
+Zulip is primarily useful for replying to people who are *outside the team* who have questions that the team can answer about Python, the Pangeo software stack, and general application questions regarding analysis.  The Team should monitor Zulip on a semi-regular basis, once or twice a day.  Fortunately, traffic on Zulip is not so great as to require more than that, even for a team our size!
+
+Zulip can also be useful for one-on-one communication between teammates.  Use Zulip for this rather than email, as much as possible.  Also, use Zulip for direct communication with another person primarily as a means of keeping other communication channels "less noisy," even if the communication specifically doesn't have anything terribly private in it.
+
+### Zulip Best Practices
+
+- Be timely in responding to people on Zulip.  One of Zulip's greatest benefits is as a "quick" communication platform.  You shouldn't expect people to respond to you (or for you to respond) *immediately*, but a person should expect getting a response within the day.
```

Comment:
Is a daily response time reasonable? Interested in others' thoughts here

Reply:
My observation is that this is the *current* response time.  Do other people agree?